### PR TITLE
Add missing translation for learning move

### LINKED ISSUE #875: Resquet for missing translation
When you teach a new move to a Pokémon, everything have the possbility to be localized, but these lines have been missed :
- "1, 2, 3 and……… Poof!"
- "And…"
If somebody is able to do it, they are located on lines 3926 and 3928 respectively in src/phases.ts

### PR DESCRIPTION
**Pull request by torranx**
Resolves #875

Translations for:
- _"1, 2, and ... ... ... Poof!"_
- _"And..."_

**Tested on local**

### CODE REVIEW

**Review by Adri1 — APPROVED** [review]

File: src/locales/de/battle.ts [hunk 1]
```diff
@@ -31,6 +31,8 @@ export const battle: SimpleTranslationEntries = {
   "learnMoveNotLearned": "{{pokemonName}} hat\n{{moveName}} nicht erlernt.",
   "learnMoveForgetQuestion": "Welche Attacke soll vergessen werden?",
   "learnMoveForgetSuccess": "{{pokemonName}} hat\n{{moveName}} vergessen.",
+  "countdownPoof": "@d{32}1, @d{15}2, @d{15}und@d{15}… @d{15}… @d{15}… @d{15}@s{pb_bounce_1}Puff!",
```
Comment (CodeTappert) [inline src/locales/de/battle.ts:34]: The correct german translation (as far as the newest mainline games are concerned) is

`Eins, zwei und... schwupp!`

Comment (torranx) [reply src/locales/de/battle.ts:34]: this is noted, thanks!

**Comment by torranx** [comment]
https://github.com/pagefaultgames/pokerogue/assets/68144167/afcffc15-ca1e-4171-8f76-59158e520d84

updated PR with correct german translation

**Review by CodeTappert — APPROVED** [review]

**Comment by Adri1** [comment]
Strange, but shoud be okay now

**Comment by Adri1** [comment]
It's a pull request to your branch

**Comment by torranx** [comment]
> It's a pull request to your branch

oh sorry let me look at it

**Comment by torranx** [comment]
nice catch, merged. thanks!

**Review by bennybroseph — APPROVED** [review]

**Pull request merged**